Dockerfiles/ApiDockerfile

```diff
@@ -3,12 +3,12 @@ RUN mkdir -p /opt/sawmill/config && \
     mkdir -p /opt/sawmill/certs && \
     openssl req -x509 -newkey rsa:2048 -nodes -keyout /opt/sawmill/certs/api.key -out /opt/sawmill/certs/api.cert -days 365 -subj "/CN=localhost" && \
     chown -R nobody:nogroup opt/sawmill
-COPY api_config.py /opt/sawmill/config/
 COPY dist/sawmill_api-*.whl /
 RUN apt update && apt upgrade -y && \
-    apt-get install -y procps && \
+    apt-get install -y procps build-essential libpq-dev && \
```

Comment:
The image is getting really chonky with this update. I'm guessing it's `build-essential` that's exploding the size 🤞. I'll look into a multi-staged build once I'm closer to trying to run/deploy this thing.